Campus-iOS/Map/MainView/PanelContent.swift

```diff
@@ -150,8 +177,6 @@ struct SearchBar: View {
         PanelContent(zoomOnUser: .constant(true),
```

Comment:
Why is the Preview actually commented out?

Reply:
Actually, all the previews here are commented out, but I don't know what they are used for nor who created/commented them out.

Reply:
They are used for the SwiftUI previews, feel free to comment all of them in again :D